test/unit/views/MovementsViewSpec.scala

```diff
@@ -113,14 +114,20 @@ class MovementsViewSpec extends ViewSpec with Injector {
       val firstDataRowElements = pageWithData.selectFirst(".govuk-table__body .govuk-table__row:nth-child(1)")
       val secondDataRowElements = pageWithData.selectFirst(".govuk-table__body .govuk-table__row:nth-child(2)")
 
+      val formatter = DateTimeFormatter.ofPattern("dd MMM YYYY 'at' HH:mm")
+
       firstDataRowElements.selectFirst(".ucr").text() mustBe validMucr
       firstDataRowElements.selectFirst(".submission-type").text() mustBe "MUCR"
-      firstDataRowElements.selectFirst(".date-of-request").text() mustBe "31 Oct 2019 at 00:00"
+      firstDataRowElements.selectFirst(".date-of-request").text() mustBe LocalDateTime
+        .of(2019, 10, 31, 0, 0)
+        .format(formatter) // "31 Oct 2019 at 00:00"
```

Comment:
Nit: I see no need for this comment. Formatter is well defined earlier.

Reply:
I had an issue with my local machine because of the original hard code String. I had a different locale that broke the test; therefore, my suggestion would be to use this approach to avoid similar problems in the future.